packages/playwright/README.md

```diff
@@ -125,84 +115,88 @@ foreach(var inapplicable in axeResults.Inapplicable)
 {
     Console.WriteLine($"Rule Id: {inapplicable.Id}.");
 }
-
 ```
 
-This method can be run on an element via the context parameter.
-This allows the inclusion and exclusion of string selectors.
-When exclude is only specified, include will default to the entire document.
-Currently the node and node lists functionality are not supported.
+`RunAxe` is supported on both Playwright `Page`s and Playwright `Locator`s. `Locator`s are the easiest way to run axe against only part of a page:
 
-``` cs
-
-AxeRunContext runContext = new AxeRunSerialContext("#my-id"); // Only run on this id.
+```cs
+ILocator locator = page.Locator("text=Sign In");
 
-runContext = new AxeRunSerialContext(null, "#my-id"); // Run on everything but this id.
+AxeResult axeResults = await locator.RunAxe();
+```
 
-runContext = new AxeRunSerialContext("button", "#my-id"); // Run on every button that does not have this id.
+Alternately, when used on a `Page`, `RunAxe` supports an `AxeRunContext` parameter which supports more complex rules for deciding which parts of a page to scan (or exclude from a scan). This corresponds to the [axe.run Context parameter](https://www.deque.com/axe/core-documentation/api-documentation/#context-parameter):
 
-runContext = new AxeRunSerialContext(new List<string>()
+```cs
+AxeRunContext runContext = new AxeRunContext()
 {
-    new List<string>()
-    {
-        "#my-frame",
-        "#my-id"
-    }
-}); // Run on the element with my-id Id and which is inside the frame with id my-frame.
-
-AxeResults axeResults = await page.RunAxe(runContext);
+    // Only run on this #my-id.
+    Include = new List<AxeSelector>() { new AxeSelector("#my-id") }
+};
 
-```
+runContext = new AxeRunContext()
+{
+    // Run on everything except #my-id.
+    Exclude = new List<AxeSelector>() { new AxeSelector("#my-id") }
+};
```

Comment:
How would you go about adding multiple `Excludes`? If I'm not mistaken this serialises to `["#my-id"]` which axe-core will treat as single selector, but any more additions will treat it as a iframe selector unless another `Exclude` entry is added

(same would apply to `Include` too)